packages/pharos-cli/src/utils/initComponentsUtil.ts

```diff
@@ -16,13 +16,22 @@ export async function addNewComponentToInitComponents(
     SyntaxKind.ImportDeclaration
   );
 
-  importStatementsDeclarations[0].addNamedImport(componentName);
+  importStatementsDeclarations.forEach((dec) => {
+    const importClause = dec.getImportClause();
+    if (importClause && importClause.getNamedImports().length > 0) {
```

Comment:
Appreciate the suggestion above @daneah . I improved the code using syntax kinda instead of elements. I too agree that if we alter the import statements order on the page, script will break. With this change, our script should add required component name exactly where we want.